deploy/07_default_strategy.ts

```diff
@@ -23,6 +34,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       },
     })
   );
+
+  const defaultStrategy = await hre.ethers.getContract("DefaultStrategy");
+  const validatorGroups = parseValidatorGroups(process.env.VALIDATOR_GROUPS);
+
+  let nextGroup = ADDRESS_ZERO;
+  for (let i = 0; i < validatorGroups.length; i++) {
+    await defaultStrategy.activateGroup(validatorGroups[i], ADDRESS_ZERO, nextGroup);
```

Comment:
If the order of these groups matter, we should at least await that the transaction is submitted onchain (i.e., obtain a TX hash) before proceeding. Otherwise the order might not be preserved

Reply:
Thanks :-)